python/valis/routes/query.py

```diff
@@ -83,6 +84,7 @@ class QueryRoutes(Base):
                  dependencies=[Depends(get_pw_db)],
                  response_model=MainSearchResponse, response_model_exclude_unset=True,
                  response_model_exclude_none=True)
+    @valis_cache()
```

Comment:
can we call the cache namespace for the query module `query`?

Reply:
Done.